### DISCUSSION TITLE
Product Icon Theme fonts cannot load

### DISCUSSION THREAD

**Discussion by dalyIsaac** [community/community#5092] [Codespaces]
Fonts associated with Product Icon Themes cannot be loaded as they violate a Content Security Policy directive: 

```
Refused to load the font 'https://miguelsolorio.vscode-unpkg.net/miguelsolorio/fluent-icons/0.0.12/extension/theme/fluent-icons.ttf' because it violates the following Content Security Policy directive: "font-src 'self' https://vscodeweb.azureedge.net/insider/ https://vscodeweb.azureedge.net/stable/".
```

![image](https://user-images.githubusercontent.com/17995621/129430633-d9cb06af-6129-49ec-a1e7-57e3a5733f73.png)

**Comment by tanmayeekamath** [comment]
Hey @dalyIsaac, thanks for reaching out. 

Can you confirm on a couple things: 
- Are you using github.dev (broswer based editor) or a full codespace instance? 
- Are you working through any firewall/network policies?

**Reply by dalyIsaac** [reply]
- I'm on github.dev
- I'm not working through any firewall/network policies
